content/en/docs/concepts/services-networking/service.md

```diff
@@ -1173,6 +1173,16 @@ SCTP is not supported on Windows based nodes.
 The kube-proxy does not support the management of SCTP associations when it is in userspace mode.
 {{< /warning >}}
 
+## Application protocols
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+The AppProtocol field provides a way to specify an application protocol to be
+used for each Service port.
```

Comment:
After reading this, I don't know what kind of values the field might take. Does this field it go in the `spec`? The `metadata`? Readers might not be able to work that out.

Reply:
Good catch, I've tried to make this more clear now, thanks!